static/components.js

```diff
@@ -11,6 +11,15 @@ import '@polymer/paper-listbox';
 import '@polymer/paper-ripple';
 import '@polymer/paper-styles/color.js';
 
+// Shoelace components
+// css is imported via _base.html in base.css, built by gulpfile.babel.js.
+import '@shoelace-style/shoelace/dist/components/button/button.js';
+import '@shoelace-style/shoelace/dist/components/dialog/dialog.js';
+import { setBasePath } from '@shoelace-style/shoelace/dist/utilities/base-path.js';
+
+// Set the base path to the folder you copied Shoelace's assets to
```

Comment:
This comment about the shoelace assets and the path used with setBasePath should be changed to whatever is appropriate for our installation.  I'm not sure yet whether the assets (images) are available this way.

Reply:
I'll fix the assets as you suggest in an upcoming PR, when I actually need the assets for something.